### PR TITLE
New gallery example for Alt/Az to Helioprojective Coordinate Transfer

### PR DESCRIPTION
**Pull request by Airmansmith97**

### CODE REVIEW

**Comment by nabobalis** [comment]
Hello @Airmansmith97! Welcome to SunPy and thank you for the example.

A general comment on the example is that could you try to change the issues that @pep8speaks highlighted? (It is kind of broken for some reason.)

File: examples/units_and_coordinates/AltAz Coordinate Transfer Final.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+
+# coding: utf-8
+
+# In[122]:
+
+
```
Comment (nabobalis) [inline examples/units_and_coordinates/AltAz Coordinate Transfer Final.py]: Lines 1-6 can be deleted.

File: examples/units_and_coordinates/AltAz Coordinate Transfer Final.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+In this example, we use balloons observations to include
+elevation as a factor.
+"""
+from astropy.coordinates import EarthLocation, AltAz, SkyCoord
+from astropy.time import Time
+from sunpy.coordinates import frames, get_sunearth_distance
+import astropy.units as u
+
+######################################################################################
+# We use AstroPy SkyCoords to define the center of the Sun
```
Comment (nabobalis) [inline examples/units_and_coordinates/AltAz Coordinate Transfer Final.py]: If you can change, AstroPy SkyCoords to AstroPy's `~astropy.coordinates.SkyCoord`. I forgot how to escape characters in markdown, but around ~astropy.coordinates.SkyCoord, you want to add a back backtick at the start and the end. This way it should  create a link to Astropy's Documentation.

**Comment by nabobalis** [comment]
Also one more comment, you could please rename the file from `AltAz Coordinate Transfer Final.py` to `AltAz_Coordinate_Transfer_.py`. That is what seems to have broken PEP8speaks.

File: examples/units_and_coordinates/AltAz Coordinate Transfer Final.py (hunk 2 above)
Comment (Airmansmith97) [reply examples/units_and_coordinates/AltAz Coordinate Transfer Final.py]: That's really convenient, thanks!

**Comment by Airmansmith97** [comment]
Can anyone explain how to label a file as 'No Changelog Entry Needed'

File: examples/units_and_coordinates/AltAz_Coordinate_Transfer_.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
 import astropy.units as u
 
 ######################################################################################
-# We use ~astropy.coordinates.SkyCoord~ to define the center of the Sun
+# We use '~astropy.coordinates.SkyCoord' to define the center of the Sun
```
Comment (Cadair) [inline examples/units_and_coordinates/AltAz_Coordinate_Transfer_.py]: this needs to be the ``` ` ``` quote not the `'` quote. :smile:

**Comment by nabobalis** [comment]
@Airmansmith97 you are the first person to encounter the new bot. So you are kind of a guinea pig here. 

Do you find the bot message opaque? Was it useful?

Since this is an addition we will require you to add a changelog entry. This goes into the changelog folder.
You will want to add a file with the name, `2656.doc.rst` and you can add just one sentence that says something like, `Added example that deals with  Alt/Az to Helioprojective Coordinate`.

File: examples/units_and_coordinates/AltAz_Coordinate_Transfer_.py (hunk 3 above)
Comment (Airmansmith97) [reply examples/units_and_coordinates/AltAz_Coordinate_Transfer_.py]: All good, appreciate it, @Cadair

**Comment by Airmansmith97** [comment]
@nabobalis Fantastic, I'm excited for the undertaking. I think in context it is very useful, I am something of a novice to this haha

File: examples/units_and_coordinates/AltAz_Coordinate_Transfer_.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+######################################################################################
+# Now we establish our location on the Earth, in this case Fort Sumner, NM.
+# We use the balloon's observational altitude as 'height'. Accuracy of 'height' is
+# far less of a concern than Lon/Lat accuracy.
+Fort_Sumner = EarthLocation(lat=34.4900*u.deg, lon=-104.221800*u.deg, height=40*u.km)
+
+######################################################################################
+# Now lets convert this to a local measurement of Altitude and Azimuth.
+
+frame_altaz = AltAz(obstime=Time(obstime), location=Fort_Sumner)
```
Comment (nabobalis) [inline examples/units_and_coordinates/AltAz_Coordinate_Transfer_.py]: Looks like `obstime` is not defined.

Comment (Airmansmith97) [reply examples/units_and_coordinates/AltAz_Coordinate_Transfer_.py]: Easy fix, thank you

**Pull request merged**